CalPatRec/src/CalHelixFinder_module.cc

```diff
@@ -338,6 +338,9 @@ namespace mu2e {
 
 	int loc = _data.nseeds[0];
 	if (loc < _data.maxSeeds()) {
+	  if (index_best == 2){
```

Comment:
I don't understand this logic.  What does index_best==2 correspond to physically?

Reply:
the case where two helices will be stored. The current diagnostic doesn't handle such a case, that's why I'm selecting the first one. This is OK as I usually run the diagnostic only with the single-particle datasets